node/src/types/block.rs

```diff
@@ -180,87 +187,41 @@ impl From<TryFromSliceError> for Error {
     }
 }
 
-/// A cryptographic hash identifying a `ProtoBlock`.
-#[derive(
-    Copy,
-    Clone,
-    DataSize,
-    Ord,
-    PartialOrd,
-    Eq,
-    PartialEq,
-    Hash,
-    Serialize,
-    Deserialize,
-    Debug,
-    Default,
-)]
-pub struct ProtoBlockHash(Digest);
-
-impl ProtoBlockHash {
-    /// Constructs a new `ProtoBlockHash`.
-    pub fn new(hash: Digest) -> Self {
-        ProtoBlockHash(hash)
-    }
-
-    /// Returns the wrapped inner hash.
-    pub fn inner(&self) -> &Digest {
-        &self.0
-    }
-}
-
-impl Display for ProtoBlockHash {
-    fn fmt(&self, formatter: &mut Formatter) -> fmt::Result {
-        write!(formatter, "proto-block-hash({})", self.0)
-    }
-}
-
 /// The piece of information that will become the content of a future block (isn't finalized or
 /// executed yet)
 ///
 /// From the view of the consensus protocol this is the "consensus value": The protocol deals with
-/// finalizing an order of `ProtoBlock`s. Only after consensus has been reached, the block's
+/// finalizing an order of `BlockPayload`s. Only after consensus has been reached, the block's
 /// deploys actually get executed, and the executed block gets signed.
 ///
 /// The word "proto" does _not_ refer to "protocol" or "protobuf"! It is just a prefix to highlight
 /// that this comes before a block in the linear, executed, finalized blockchain is produced.
```

Comment:
```suggestion
```

Reply:
Good catch! Fixed in https://github.com/casper-network/casper-node/pull/1290/commits/74aafadcaf6484ad8fcd2dcb7065349869471683.